src/agents/evidence/writeBundle.ts

```
@@ -0,0 +1,158 @@
+      } else {
+        acc[key] = redactValue(entry);
+      }
+      return acc;
+    }, {});
+  }
+  return value;
+};
+
+const assertNoTimestamps = (value: unknown, pathLabel = ''): void => {
+  if (Array.isArray(value)) {
+    value.forEach((entry, index) =>
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)